app/src/test/java/io/github/droidkaigi/confsched2017/util/AssetsUtilTest.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.droidkaigi.confsched2017.util
+
+import com.google.firebase.FirebaseApp
+import com.taroid.knit.should
+import org.junit.Assert.fail
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import java.lang.reflect.InvocationTargetException
+
+@RunWith(RobolectricTestRunner::class)
+class AssetsUtilTest {
+    @Test
+    @Throws(Exception::class)
+    fun ctor() {
+        try {
+            val ctor = AssetsUtil::class.java.getDeclaredConstructor()
```

Comment:
https://github.com/DroidKaigi/conference-app-2017/blob/master/app/src/test/java/io/github/droidkaigi/confsched2017/util/DateUtilTest.kt
Because DateUtilTest has constructor's test, I did it.
Constructor test is unnecessary?